pkg/volume/csi/nodeinfomanager/nodeinfomanager.go

```diff
@@ -112,6 +112,7 @@ func (nim *nodeInfoManager) InstallCSIDriver(driverName string, driverNodeID str
 	}
 
 	nodeUpdateFuncs := []nodeUpdateFunc{
+		removeMaxAttachLimit(driverName), // remove in 1.34
```

Comment:
Please update the comment with a reason *why* should it be removed in 1.34.

Reply:
updated.